### DISCUSSION TITLE
Is it possible to format import groups alphabetically and always in new lines?

### DISCUSSION THREAD

**Discussion by manuelpoelzl** [biomejs/biome#2587] [Q&A]
Hey folks,

I'm new to biome and I wonder if it is possible, to have import groups sorted alphabetically?
For example, currently I do have this import group:

```tsx

import globalSlice, {
	setIsCustomAppAccessFlowLocked,setCustomAppAccessLoaded, setIsCustomAppAccessDirty
} from "../../../../../../features/Globals/GlobalSlice";

```

The biome.json file looks like this:

```json
{
  "$schema": "https://biomejs.dev/schemas/1.7.0/schema.json",
  "organizeImports": {
    "enabled": true
  },
  "linter": {
    "enabled": false,
    "rules": {
      "recommended": true
    }
  },
  "formatter": {
    "lineWidth": 155,
    "enabled": true
  }
}
```

I'm using this command to invoke biome `npx @biomejs/biome check --apply ./src`

As soon as biome has processed the files the import group looks like this:

```tsx
import globalSlice, {
	setIsCustomAppAccessFlowLocked,
	setCustomAppAccessLoaded,
	setIsCustomAppAccessDirty,
} from "../../../../../../features/Globals/GlobalSlice";
```

Multiline is working perfectly fine, however as you can see, the imports are not sorted alphabetically, which would be nice.
Is there any configuration option in biome which I can use to achive this?

Thanks in advance!

**Comment by unvalley** [comment]
Sorry for the delay in commenting. Currently the organizeImport doesn't have the feature (and options) to sort alphabetically.

**Reply by unvalley** [reply]
You may be interested in this:
https://github.com/biomejs/biome/discussions/3015

**Comment by ematipico** [comment]
Closing: https://github.com/biomejs/biome/discussions/3015
